x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/create_connector_flyout/header.tsx

```diff
@@ -47,6 +51,25 @@ const FlyoutHeaderComponent: React.FC<{
               <EuiText size="s" color="subdued">
                 {actionTypeMessage}
               </EuiText>
+              {featureIds && featureIds.length > 0 && (
+                <>
+                  <EuiSpacer size="m" />
+                  <EuiFlexGroup
+                    data-test-subj="create-connector-flyout-header-availability"
+                    wrap
+                    responsive={false}
+                    gutterSize="xs"
+                    alignItems="center"
+                  >
+                    Availability:{' '}
```

Comment:
Note to use `<FormattedMessage ... />` on `Availability` for localization purposes.